src/test/java/com/ibm/cloud/sdk/core/test/service/ErrorResponseTest.java

```diff
@@ -73,25 +74,74 @@ public void setUp() throws Exception {
     service.setServiceUrl(getMockWebServerUrl());
   }
 
+  private void verifyException(ServiceResponseException e, int expectedStatusCode, String expectedMessage,
+      String expectedResponseBody) {
+
+    assertEquals(e.getStatusCode(), expectedStatusCode);
+    assertEquals(e.getMessage(), expectedMessage);
+    assertEquals(e.getResponseBody(), expectedResponseBody);
+  }
+
   /**
    * Test HTTP status code 400 (Bad Request) error response.
    */
   @Test
   public void testBadRequest() {
 
     String message = "The request failed because the moon is full.";
+    String responseBody = "{\"error\": \"" + message + "\"}";
```

Comment:
Changed each of the test methods to initialize `requestBody` so that it can also be used as an expected value.